src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -170,6 +170,17 @@ class OPENASSETIO_CORE_EXPORT Manager {
    */
   void initialize(InfoDictionary managerSettings);
 
+  /**
+   * Clears any internal caches.
+   *
+   * Only applicable if the manager makes use of any caching, otherwise
+   * it is a no-op.  In caching interfaces, this should cause any
+   * retained data to be discarded to ensure future queries are fresh.
+   *
+   * @unstable
```

Comment:
[question] Is `flushCaches` still `@unstable`?

Reply:
I don't think so now we've decided to keep it.

Reply:
Removed and force-pushed: https://github.com/OpenAssetIO/OpenAssetIO/compare/5c68a0cadcf4b23ee00a9a826675c7580c194e21..15e96ff36b2b862da190a3e7dbcfb913ae007d92